apps/console/src/extensions/components/authenticators/sms-otp/sms-otp-authenticator.tsx

```diff
@@ -16,13 +16,15 @@
  * under the License.
  */
 
+import { AppState } from "apps/console/src/features/core/store";
```

Comment:
Shall we please add the relative path here?

Reply:
Fixed with 32df88f9368664421e57f487e6477045fa17c5d6